setup.cfg

```diff
@@ -1,6 +1,85 @@
-[bdist_rpm]
-packager = Sylvain Thenault  <sylvain.thenault@logilab.fr>
-provides = pylint
+[metadata]
+name = pylint
+license = GPL-2.0-or-later
+license_file = COPYING
```

Comment:
`license_file` is deprecated and has been replace by `license_files`.
I would also suggest to add `CONTRIBUTORS.txt` here as well. What do you think?

Reply:
👍